chain/chunks/src/lib.rs

```diff
@@ -44,10 +45,11 @@ mod test_utils;
 mod types;
 
 const CHUNK_PRODUCER_BLACKLIST_SIZE: usize = 100;
-const CHUNK_REQUEST_RETRY_MS: u64 = 100;
-const CHUNK_REQUEST_SWITCH_TO_OTHERS_MS: u64 = 400;
-const CHUNK_REQUEST_SWITCH_TO_FULL_FETCH_MS: u64 = 3_000;
+const CHUNK_REQUEST_RETRY_MS: u64 = 500;
+const CHUNK_REQUEST_SWITCH_TO_OTHERS_MS: u64 = 1_000;
+const CHUNK_REQUEST_SWITCH_TO_FULL_FETCH_MS: u64 = 4_000;
```

Comment:
`chunks_recovered_from_full` will likely fail now.

Reply:
I took a stab at a fix in https://github.com/nearprotocol/nearcore/pull/2769/commits/54c6a86343154c76182a5ebba57c09e496d44d3a

I put the times in terms of what I thought were the relevant constants. Let me know what you think.